pygmt/tests/test_clib.py

```diff
@@ -873,16 +873,22 @@ def mock_defaults(api, name, value):  # pylint: disable=unused-argument
         """
         Put 'bla' in the value buffer.
         """
-        value.value = b"bla"
+        if name == b"API_VERSION":
+            value.value = b"1.2.3"
+        else:
+            value.value = b"bla"
         return 0
 
     ses = clib.Session()
     ses.create("test-session")
     with mock(ses, "GMT_Get_Default", mock_func=mock_defaults):
         # Check for an empty dictionary
         assert ses.info
-        for value in ses.info.values():
-            assert value == "bla"
+        for key, value in ses.info.items():
+            if key == "version":
+                assert value == "1.2.3"
+            else:
+                assert value == "bla"
```

Comment:
Should there also be a mock for `if name == b"API_BIN_VERSION"` and `if key == "binary_version"`? Or would that break with #2079?

Reply:
At line 173 in `pygmt/clib/session.py`, we have the following code:
```
            if Version(self._info["version"]) >= Version("6.4.0"):
                self._info["binary version"] = self.get_default("API_BIN_VERSION")
```
The `Version()` class requires `self._info["version"]` being a valid version string, so we can't give a value like `bla` to `API_VERSION`.

There is no such problem for `API_BIN_VERSION`, so no special operation is done for it.